charmhelpers/contrib/storage/linux/ceph.py

```diff
@@ -1482,13 +1482,21 @@ def send_request_if_needed(request, relation='ceph'):
             relation_set(relation_id=rid, broker_req=request.request)
 
 
+def has_broker_rsp(rid=None, unit=None):
+    rdata = relation_get(rid=rid, unit=unit) or {}
+    broker_rsp = rdata.get(get_broker_rsp_key())
+    if not broker_rsp:
```

Comment:
It's generally nicer to work with positives:  i.e.

```python
    return True if broker_rsp else False
```

Reply:
Thanks!